### ISSUE TITLE
Stylelint not reporting errors on nested css templates.

### ISSUE THREAD

**Issue by bob-difronzo** [hudochenkov/postcss-styled-syntax#7]
When running Stylelint on the following code with `"unit-allowed-list": ["rem"]` set, it reports no errors instead of `✖  Unexpected unit "px"`:
```js
const StyledDiv = styled.div`
  ${css`
    padding: 1px;
  `}
`;
```

It only seems to happen when the `css` tagged template is used *within* the interpolation of another tagged template because this does report the correct error:
```js
const StyledDiv = styled.div(
  () => css`
    padding: 1px;
  `
);
```

Here are the versions I am using:
* `postcss-styled-syntax@0.3.1`
* `stylelint@15.1.0`
* `styled-components@5.3.6`
* `node@18.14.0`
* `npm@8.18.3`

<details>
<summary>

## Full reproduction

</summary>

<details open>
<summary>

### `package.json`

</summary>

```json
{
  "name": "styled-syntax-nested-issue",
  "version": "1.0.0",
  "private": true,
  "type": "module",
  "main": "index.js",
  "scripts": {
    "lint": "stylelint index.js"
  },
  "license": "ISC",
  "dependencies": {
    "styled-components": "5.3.6"
  },
  "devDependencies": {
    "postcss-styled-syntax": "0.3.1",
    "stylelint": "15.1.0"
  }
}
```

</details>

<details open>
<summary>

### `.stylelintrc.json`

</summary>

```json
{
  "customSyntax": "postcss-styled-syntax",
  "rules": {
    "unit-allowed-list": ["rem"]
  }
}
```

</details>

<details open>
<summary>

### `index.js`

</summary>

```js
import styled, { css } from 'styled-components';

const someCondition = Math.random() > 0.5;

const Works1 = styled.div`
  padding: 1px;
`;

const Works2 = styled.div(
  () => css`
    padding: 1px;
  `
);

const Works3 = styled.div(() =>
  someCondition
    ? css`
        padding: 1px;
      `
    : css`
        padding: 2px;
      `
);

const DoesNotWork1 = styled.div`
  ${css`
    padding: 1px;
  `}
`;

const DoesNotWork2 = styled.div`
  ${
    someCondition
      ? css`
        padding: 1px;
      `
      : css`
        padding: 2px;
      `
  }
`;

const DoesNotWork3 = styled.div(
  () =>
    css`
      ${
        someCondition
          ? css`
            padding: 1px;
          `
          : css`
            padding: 2px;
          `
      }
    `
);

const DoesNotWork4 = styled.div`
  ${() =>
    css`
      padding: 1px;
    `}
`;
```

</details>

<details open>
<summary>

### `.vscode/settings.json`

</summary>

```json
{
  "stylelint.validate": ["javascript"]
}
```

</details>
</details>

**Comment by hudochenkov** [comment]
Thank you for a detailed report!

This was a terrible issue. Since release of this package Stylelint couldn't lint inside `css` interpolations 🤦 

I fixed the problem in `0.3.2`.

**Comment by bob-difronzo** [comment]
Thank you for the quick response! That's all working now. 😄
